arches_references/src/arches_references/components/tree/PresentationControls.vue

```diff
@@ -14,15 +14,16 @@ import {
 import { shouldUseContrast } from "@/arches_references/utils.ts";
 
 import type { Ref } from "vue";
-import type { Language } from "@/arches/types";
+import type { Language } from "@/arches_vue_utils/types";
 
 const { $gettext } = useGettext();
 
 const selectedLanguage = inject(selectedLanguageKey) as Ref<Language>;
 
-const { expandAll, collapseAll } = defineProps<{
+const { expandAll, collapseAll, fontFamily } = defineProps<{
     expandAll: () => void;
     collapseAll: () => void;
+    fontFamily: string;
```

Comment:
if font-family is only being passed through this component, why bother defining it on a higher level?

Reply:
This component is the first test case for trying to genericize a component enough to pull it out into arches-vue-utils. So this is parameterized because we were (at one point?) using slightly different font-family values in the two apps.

Should we strive to use the same? Probably. But one of them needs to blend in with core arches chrome and the other doesn't. So IDK.

Should we find a different pattern here like relying on font-family: inherit/unset/revert/whatever on the `pt` options so that this component doesn't need have this parameter? Probably! I can take a little look. 👍

Reply:
I can also just back out this change so we don't block this PR on it?

Reply:
yah I'd say back out for now to not accrue tech debt and we cross that bridge when we need to 👍